{% extends "layout.html" %}
  
{% block title %}Project Description{% endblock %}
{% block lead %}Describe how and why you chose your iGEM project.{% endblock %}

{% block page_content %}

<div class="row mt-4">
  <div class="col">
    <div class="bd-callout bd-callout-info">
      <h4>Bronze Medal Criterion #3</h4>
      <p>Describe how and why you chose your iGEM project.<p>
      <hr>
      <p>Please see the <a href="https://competition.igem.org/judging/medals">2023 Medals Page</a> for more information.</p>
    </div>
  </div>
</div>

<div class="row mt-4">
  <h2>Project description</h2>
  <hr>
  <p><b>Today in Denmark, the question is not where PFAS is, but where PFAS isn't.</b> Recent advances in environmental and health sciences have confirmed that PFAS, a class of "forever chemicals", are extremely detrimental to planetary and human well-being. As efforts are made to identify sites of contamination and sources of pollution, detection methods are not streamlined or efficient. A major obstacle in effectively controlling PFAS lies in sampling bias, where the need for specialized equipment, limited resources, and trained personnel hinders the acquisition of accurate, high-resolution, real-time information on global PFAS concentrations. </p>
  <p>That is what we wish to target with our project, FluoroLoop, by designing and validating a cutting edge molecular diagnostic device that will make quick, easy, and cheap detection of PFAS a reality. Our overarching goal is to establish a comprehensive system for PFAS detection, followed by centralized data collection, as illustrated in Figure 1. Through this initiative, we aspire to significantly enhance our understanding of PFAS contamination and contribute to the well-being of both the planet and human health.</p>
  <img src="PFOA4b.png">
  <h3>Persistent pollution and the problems of PFAS</h3>
  <p>Since the 1940s, per- and polyfluorinated alkyl substances (PFAS) have been produced for a myriad of purposes across the world. As fully (per-) or highly (poly-) fluorinated compounds, PFAS show exceptionally high stability due to the carbon-fluorine bonds and often have an amphiphilic nature conferred by a functional group. Their range of applications span from everyday products like water-proof clothing, cosmetics, and Teflon pans to industrial uses such as pesticides and firefighting foam (Brunn et al. 2023).   </p>
  <p>However, the features that make PFAS extraordinary also make them extraordinarily problematic. For the majority of PFAS, incineration temperatures must exceed 1100°C to achieve full mineralization of the strong carbon-fluorine bond. Similarly, some microorganisms can degrade the non-fluorine parts of polyfluorinated compounds, but none are able to cleave the fluorine bonds. This means that PFAS are rarely broken down fully, neither in industrial waste nor in nature, leading to long-term environmental pollution. In addition to their persistence, many PFAS molecules are associated with bioaccumulation and various severely harmful biological effects. The list of consequences in humans includes among many other things immunotoxicity, decreased fertility, and increased risk of multiple types of cancer. Despite this, we continue producing and using PFAS products and globally, it is estimated that we produce 320,000 tons of fluoropolymers annually (Brunn et al. 2023). </p>
  <p>In the summer of 2021, PFAS became a hot topic in Danish news as the limit value for a subset of PFAS in drinking water was lowered. It became clear that many of our water reservoirs contain shockingly high levels of PFAS, far beyond what is considered safe for consumption. With more than 70% of measured water sources already containing PFAS, there is an increased threat to our future supply of clean drinking water (DR 14/03/2023). Following this, a media storm ensued, reporting PFAS pollution in free-roaming cattle, organic eggs, popular swimming areas, everywhere. The ubiquitous nature of these pollutants has not only sparked fear in the population, but also caused Denmark to advocate a PFAS ban throughout the EU. However, as part of the fight against PFAS pollution, it is crucial to identify high risk areas and gain an increased resolution of the problem.</p>
  <h3>Current detection methods & challenges</h3>
  <p>The current method of choice for PFAS detection and identification consists of sample enrichment followed by analysis using liquid chromatography and mass spectrometry (LCMS). While the method provides a sensitive limit of detection at 1 ng/L for many PFAS, it requires extensive training to perform, and a single sample typically cost hundreds of dollars to analyze (Cordner et al. 2021). Additionally, samples must be brought to a laboratory for analysis, resulting in a slow process. </p>
  <p>What we need instead is a cheap, quick, and easy-to-use detection method.</p>
  <h3>Our solution: a modular, cell-free PFOA biosensor</h3>
  <p>To address this problem, we decided on our project: FluoroLoop. Our goal is to integrate a tRNA-mimicking structure (TMS) with a PFOA-specific aptamer to create a novel, cell-free biosensor. The TMS construct is a trans-acting gene regulator, and we aim to validate it as a modular biobrick, which can be converted to detect various compounds, most importantly other environmental pollutants. Ideally, we want to incorporate the PFAS biosensor in a cell-free system, which can be used as a low-cost rapid test kit for a citizen-driven effort to pinpoint PFAS pollution around the world. In parallel to our wet lab efforts, we plan to construct a bioinformatics tool to streamline and optimize <i>in silico</i> aptamer design. </p>
  <h3>The biology behind: tRNA-mimicking structures and aptamers</h3>
  <p>We will begin the project by validating the TMS and corresponding protocol from Paul et al. (2020) with multiple ligands. Following this, we will integrate the PFOA-specific aptamers developed by Park et al. (2022) into the D-loop module of the TMS in various configurations to investigate which constructs perform best in our system.</p>
  <p>The benefit of a TMS-based molecular device compared to many others, e.g., riboswitches, is that the TMS system is trans-encoded. This is beneficial as it allows for modular design and does not require upstream modifications of the mRNA in question. The TMS is based on bacterial tRNA, which has been modified to bind both flanking sites of the ribosome binding site with a repressor domain to achieve tight control of gene expression. An anti-repressor is then used to break the repression by binding to a matching site in the D-loop, giving a dose-dependent output in relation to the anti-repressor concentration. By switching the D-loop to a complementary sequence or aptamer, the TMS can be customized to respond to a range of ligands, including proteins and small molecules (Paul et al. 2020). Reconstruction of such a system will result in a valuable and easily adaptable BioBrick that can be used to detect any molecule for which an aptamer exists. In our case, the main goal is to validate the system and insert a PFOA-sensing aptamer, as illustrated in Figure X.</p>
  <img src="/wiki/pages/images/PFOA5b.png" alt="Diagram of our biosensor">
  <p>Aptamers, such as the one mentioned above, are single-stranded DNA or RNA molecules, capable of selectively binding proteins, peptides, carbohydrates, small molecules, toxins, or even live cells. Since accumulation of PFAS in the environment is recent anthropogenic problem, the evolution of natural detection of these fluorinated compounds is highly limited. However, through the process of systematic evolution of ligands by exponential enrichment (SELEX), a library of aptamers can be tested for their affinity to specific molecules (Park et al. 2022). In Park et al. (2022), SELEX has been applied to discover aptamers specific for PFOA, using ssDNA. </p>
  <h3>Bioinformatics for better aptamer selection</h3>
  <p>In addition to <i>in vitro</i> validation, our plan also involves the development of a bioinformatics tool. The purpose of the tool is to streamline the design and optimization of aptamers by consolidating various molecular modeling tools used for aptamer dynamics into a single pipeline. By providing a comprehensive understanding of aptamer-analyte dynamics prior to laboratory experimentation, we anticipate that this tool will enhance the attractiveness of our molecular device.</p>
  <p>Usually, aptamers are obtained through the challenging and time-consuming process SELEX, as mentioned earlier. Unfortunately, SELEX is experimentally challenging, time consuming, and does not guarantee the discovery of the best-performing aptamer (Flamme et al. 2019, Zhu et al. 2019). To address this limitation, researchers have been exploring predictive tools that take a rational approach to identifying the most effective aptamer sequence for a given ligand (Emami et al. 2020).</p>
  <p>Notably, the Heidelberg iGEM team developed a tool called Making Aptamers Without SELEX (MAWS) in 2015, which was subsequently improved in 2017 (iGEM Heidelberg 2015, iGEM Heidelberg 2017). MAWS utilizes an entropic criterion for progressive selection of bases to predict aptamers <i>in silico</i>.</p>
  <p>However, there is still limited understanding of the conformational changes that occur in aptamers upon ligand binding. This knowledge is crucial for a comprehensive characterization of intracellular aptamer behavior and the refinement of models. Additionally, there is a lack of integrated pipelines that encompass all the steps involved in modeling aptamer structure prediction, docking, and molecular dynamics. To address these challenges, our objective is to construct a holistic aptamer design pipeline utilizing workflow managers such as Docker or Snakemake.</p>
  <h3>Designing a cell free system for the PFOA quick test</h3>
  <p>In the end, our vision is to integrate the PFAS biosensor in a cell-free system for future distribution based on Guzman-Chavez et al. (2022). Cell-free technology utilizes in vitro transcription-translation systems in conjunction with cell extracts to provide ribosomes and transcription factors. Additionally, various other components are necessary for protein production. Although this technology has been financially out of reach for many applications, recent advancements offer promising prospects for significant cost reduction. These advancements encompass the utilization of more affordable energy sources, optimizing production with low-cost energy inputs, and simplifying the reaction mixture (Guzman-Chavez et al. 2022). Given our aim of providing an affordable PFAS rapid test, it is imperative to integrate it into a cost-effective cell-free system.</p>
  <h3>Citizen science and environmental protection</h3>
  <p>With the goal of developing a cell-free detection system for PFAS quantification, we aim to engage citizens across Denmark, and hopefully globally, in the process of pinpointing PFAS pollution. By creating an affordable, fast, and user-friendly device, we aim to empower citizens to gain comprehensive insights into the environmental conditions of their surroundings. This will enable individuals to actively monitor their own exposure levels and enhance the general understanding of the extent of PFAS pollution.</p>
  <p>Additionally, through citizen science reporting, we hope to gather data on PFAS concentrations worldwide, creating a global heat map that identifies PFAS hotspots. Participants will receive test kits to examine their local areas. They can then upload their results to our website, where the data will be collected and presented on a user-friendly global map interface. This collective effort will raise public awareness about PFAS contamination while providing researchers with more detailed information on the distribution and movement of PFAS. Such knowledge can support the development of effective containment and control measures.</p>
</div>

<div class="row mt-4">
  <h2>References</h2>
  <ul>
    <li>Brunn, H., Arnold, G., Körner, W., Rippen, G., Steinhäuser, K. G., & Valentin, I. (2023). PFAS: forever chemicals—persistent, bioaccumulative and mobile. Reviewing the status and the need for their phase out and remediation of contaminated sites. Environmental Sciences Europe 2023 35:1, 35(1), 1–50. https://doi.org/10.1186/S12302-023-00721-8</li>
    <li>Cordner, A., Goldenman, G., Birnbaum, L. S., Brown, P., Miller, M. F., Mueller, R., Patton, S., Salvatore, D. H., & Trasande, L. (2021). The True Cost of PFAS and the Benefits of Acting Now. Environmental Science and Technology, 55(14), 9630–9633. https://doi.org/10.1021/ACS.EST.1C03565/ASSET/IMAGES/LARGE/ES1C03565_0001.JPEG</li>
    <li>Emami, N., Pakchin, P. S., & Ferdousi, R. (2020). Computational predictive approaches for interaction and structure of aptamers. Journal of Theoretical Biology, 497, 110268. https://doi.org/10.1016/J.JTBI.2020.110268</li>
    <li>Flamme, M., McKenzie, L. K., Sarac, I., & Hollenstein, M. (2019). Chemical methods for the modification of RNA. Methods, 161, 64–82. https://doi.org/10.1016/J.YMETH.2019.03.018</li>
    <li>Guzman-Chavez, F., Arce, A., Adhikari, A., Vadhin, S., Pedroza-Garcia, J. A., Gandini, C., Ajioka, J. W., Molloy, J., Sanchez-Nieto, S., Varner, J. D., Federici, F., & Haseloff, J. (2022). Constructing Cell-Free Expression Systems for Low-Cost Access. ACS Synthetic Biology, 11(3), 1114–1128. https://doi.org/10.1021/ACSSYNBIO.1C00342/SUPPL_FILE/SB1C00342_SI_002.XLSX</li>
    <li>Park, J., Yang, K. A., Choi, Y., & Choe, J. K. (2022). Novel ssDNA aptamer-based fluorescence sensor for perfluorooctanoic acid detection in water. Environment International, 158, 107000. https://doi.org/10.1016/J.ENVINT.2021.107000</li>
    <li>Paul, A., Warszawik, E. M., Loznik, M., Boersma, A. J., & Herrmann, A. (2020). Modular and Versatile Trans-Encoded Genetic Switches. Angewandte Chemie International Edition, 59(46), 20328–20332. https://doi.org/10.1002/ANIE.202001372</li>
    <li>Se interaktivt kort med de 104 største PFAS-forureninger af grundvandet | Indland | DR. (n.d.). Retrieved June 27, 2023, from https://www.dr.dk/nyheder/indland/se-interaktivt-kort-med-de-104-stoerste-pfas-forureninger-af-grundvandet</li>
    <li>Team:Heidelberg/software/maws - 2015.igem.org. (n.d.). Retrieved June 27, 2023, from https://2015.igem.org/Team:Heidelberg/software/maws</li>
    <li>Team:Heidelberg/Software/MAWS - 2017.igem.org. (n.d.). Retrieved June 27, 2023, from https://2017.igem.org/Team:Heidelberg/Software/MAWS</li>
    <li>Zhu, C., Yang, G., Ghulam, M., Li, L., & Qu, F. (2019). Evolution of multi-functional capillary electrophoresis for high-efficiency selection of aptamers. Biotechnology Advances, 37(8), 107432. https://doi.org/10.1016/J.BIOTECHADV.2019.107432 </li>
  </ul>
</div>

<div class="row mt-4">
  <div class="col-lg-8">
    <h2>What should this page contain?</h2>
    <hr>
    <ul>
      <li>A clear and concise description of your project.</li>
      <li>A detailed explanation of why your team chose to work on this particular project.</li>
      <li>References and sources to document your research.</li>
      <li>Use illustrations and other visual resources to explain your project.</li>
    </ul>
  </div>
  <div class="col-lg-4">
    <h2>Inspirations</h2>
    <hr>
    <ul>
      <li><a href="https://2022.igem.wiki/dtu-denmark/description">2022 DTU-Denmark</a></li>
      <li><a href="https://2019.igem.org/Team:ITESO_Guadalajara/Description">2019 ITESO Guadalajara</a></li>
      <li><a href="https://2020.igem.org/Team:Technion-Israel/Description">2020 Technion Israel</a></li>
      <li><a href="https://2020.igem.org/Team:Botchan_Lab_Tokyo/Description">2020 Botchan Lab Tokyo</a></li>
      <li><a href="https://2020.igem.org/Team:St_Andrews/Description">2020 St Andrews</a></li>
      <li><a href="https://2020.igem.org/Team:MIT/Description">2020 MIT</a></li>
    </ul>
  </div>
</div>

<div class="row mt-4">
  <div class="col-lg-8">
    <h2>Some advice</h2>
    <hr>
    <p>We encourage you to put up a lot of information and content on your wiki, but we also encourage you to include summaries as much as possible. If you think of the sections in your project description as the sections in a publication, you should try to be concise, accurate, and unambiguous in your achievements. Your Project Description should include more information than your project abstract.</p>
  </div>
  <div class="col-lg-4">
    <h2>References</h2>
    <hr>
    <p>iGEM teams are encouraged to record references you use during the course of your research. They should be posted somewhere on your wiki so that judges and other visitors can see how you thought about your project and what works inspired you.</p>
  </div>
</div>

{% endblock %}
