{% extends "layout.html" %}

{% block page_content %}
<div class="page">
  <div class="page-header">
    <h1>Description</h1>
  </div>
  <div class="container page-content">
    <div class="row h-100">
      <div class="toc col-lg-5">
        <h5>Menu</h5>
        <ul>
          <li>
            <a href="#project_description"> > Project Description</a>
            <ul>
              <li><a href="#pfas_problem">> The PFAS problem</a></li>
              <li><a href="#current_method">> Current detection methods</a></li>
              <li><a href="#our_solution">> Our solution</a></li>
              <li><a href="#biology_behind">> The biology behind</a></li>
              <li><a href="#bioinformatics">> Bioinformatics</a></li>
              <li><a href="#cell_free">> Cell free system</a></li>
              <li><a href="#high_turnover">> High-turnover screening</a></li>
              <li><a href="#conclusion">> Conclusion</a></li>
            </ul>
          </li>
          <li>
            <a href="#references"> > References</a>
          </li>
        </ul>
        <svg class="toc-marker" width="200" height="200" xmlns="http://www.w3.org/2000/svg">
          <path stroke="#fff" stroke-width="3" fill="transparent" stroke-dasharray="0, 0, 0, 1000" stroke-linecap="round" stroke-linejoin="round" transform="translate(-0.5, -0.5)" />
        </svg>  
      </div>
      <div class="page-text col-lg-9">
        <h2 id="project_description">Project Description</h2>
        <hr>
        <h3 id="pfas_problem">The PFAS problem</h3>     
        <p><strong>Our journey begins amidst news stories and public health anxiety, as more and more reports about the “forever chemicals”, collectively known as PFAS, rings around the Copenhagen area.</strong></p>
        <p>For decades these chemicals went under the radar. Given that their negative effects were not known, and that they were remarkably stable and hydrophobic, they were increasingly used in the production of non-stick pans, raincoats, and firefighting foams. However, these positive attributes cut both ways – due to their inherent stability, thanks to the C-F bonds present in their structure, these molecules are practically indestructible. This means that when these molecules are polluted into the environment, they simply do not degrade. This stability is rendered dangerous by the fact that PFAS has been linked to cancer, infertility, and a host of other health problems in humans and for the environment. Clearly, we have a problem – indestructible, dangerous compounds have been steadily leaking into the environment. This is a problem we hope to help solve through the development of our project – FluoroLoop.</p>
        <p>The need to clean up the environment of PFAS molecules is clear, but this means that we first need to know where this compound is. This is the specific niche our iGEM team wanted to approach using synthetic biology.</p>
        <p>We want to make quick, easy, and cheap detection of PFAS a reality. Our overarching goal is to establish the foundation for a comprehensive system for PFAS detection, followed by centralized data collection, as illustrated in Figure 1. Through this initiative, we aspire to significantly enhance our understanding of PFAS contamination and contribute to the well-being of both the planet and human health.</p>
        <p>FIGURE 1</p>
        <p>Since the 1940s, per- and polyfluorinated alkyl substances (PFAS) have been produced for a myriad of purposes across the world. As fully (per-) or highly (poly-) fluorinated compounds, PFAS show exceptionally high stability due to the carbon-fluorine bonds and often have an amphiphilic nature conferred by a functional group. Their range of applications span from everyday products like water-proof clothing, cosmetics, and Teflon pans to industrial uses such as pesticides and firefighting foam (Brunn et al. 2023).
        However, the features that make PFAS extraordinary also make them extraordinarily problematic. For the majority of PFAS, incineration temperatures must exceed 1100°C to achieve full mineralization of the strong carbon-fluorine bond. Similarly, some microorganisms can degrade the non-fluorine parts of polyfluorinated compounds, but none are able to cleave the fluorine bonds. This means that PFAS are rarely broken down fully, neither in industrial waste nor in nature, leading to long-term environmental pollution. In addition to their persistence, many PFAS molecules are associated with bioaccumulation and various severely harmful biological effects. The list of consequences for humans includes among many other things immunotoxicity, decreased fertility, and increased risk of multiple types of cancer. Despite this, we continue producing and using PFAS products and globally, it is estimated that we produce 320,000 tons of fluoropolymers annually (Brunn et al. 2023).</p>
        <p>In the summer of 2021, PFAS became a hot topic in Danish news as the limit value for a subset of PFAS in drinking water was lowered. It became clear that many of our water reservoirs contain shockingly high levels of PFAS, far beyond what is considered safe for consumption. With more than 70% of measured water sources already containing PFAS, there is an increased threat to our future supply of clean drinking water (DR 14/03/2023). Following this, a media storm ensued, reporting PFAS pollution in free-roaming cattle, organic eggs, popular swimming areas, everywhere. The ubiquitous nature of these pollutants has not only sparked fear in the population, but also caused Denmark to advocate a PFAS ban throughout the EU. However, as part of the fight against PFAS pollution, it is crucial to identify high risk areas and gain an increased resolution of the problem.</p>
        <h3 id="current_method">Current detection methods &amp; challenges</h3>
        <p>The current method of choice for PFAS detection and identification consists of sample enrichment followed by analysis using liquid chromatography and mass spectrometry (LCMS). While the method provides a sensitive limit of detection at 1 ng/L for many PFAS, it requires extensive training to perform, and a single sample typically costs hundreds of dollars to analyze (Cordner et al. 2021). Additionally, samples must be brought to a laboratory for analysis, resulting in a slow process. Furthermore, these resources are not available in all parts of the world, making current PFAS testing standards non-equitable. Thus, there are 3 clear problems with current PFAS testing:</p>
        <ol>
        <li><strong>Cost</strong> – LC-MS is costly for high throughput sampling and resource intensive.</li>
        <li><strong>Strong</strong> – The process of getting samples in the field and bringing them back to the lab is laborious and not efficient.</li>
        <li><strong>Non-equitable</strong> – Analytical machinery and the availability of trained personnel are things that are present in only certain parts of the world. This means that reliable PFAS testing is only available for a fraction of the globe.</li>
        </ol>
        <p>What we need instead is a cheap, quick, and easy-to-use detection method that can be used anywhere in the world. </p>
        <h3 id="our_solution">Our solution: a modular PFOA biosensor</h3>
        <p>To address this problem, we decided on our project: FluoroLoop. Our goal is to integrate a tRNA-mimicking structure (TMS) with a PFOA-specific aptamer to create a novel molecular biosensor. The TMS construct is a trans-acting gene regulator, and we aim to validate it as a modular biobrick, which can be converted to detect various compounds, such as PFOA and other environmental pollutants. Ideally, we want to validate this PFAS biosensor for future implementation in a cell-free system, which can be used as a low-cost rapid test kit to help pinpoint PFAS pollution around the world. In parallel to our wet lab efforts, we constructed a bioinformatics tool to streamline and optimize in silico aptamer design.</p>
        <h3 id="biology_behind">The biology behind: tRNA-mimicking structures and aptamers</h3>
        <p>We will begin the project by validating the TMS and corresponding protocol from Paul et al. (2020) with multiple ligands. Following this, we integrated the PFOA-specific aptamers developed by Park et al. (2022) into the D-loop module of the TMS in various configurations to investigate which constructs perform best in our system. Alongside the tests of the PFOA aptamers, we also tested different known aptamers, specifically two versions of a manganese aptamer from Dambach et al. (2015), and two versions of a theophylline aptamer from Suess et al. (2004).
        The benefit of a TMS-based molecular device compared to many others, e.g., riboswitches, is that the TMS system is trans-encoded. This is beneficial as it allows for modular design and does not require upstream modifications of the mRNA in question. The TMS is based on bacterial tRNA, which has been modified to bind both flanking sites of the ribosome binding site with a repressor domain to achieve tight control of gene expression. An anti-repressor is then used to break the repression by binding to a matching site in the D-loop, giving a dose-dependent output in relation to the anti-repressor concentration. By switching the D-loop to a complementary sequence or aptamer, the TMS can be customized to respond to a range of ligands, including proteins and small molecules (Paul et al. 2020). Reconstruction of such a system will result in a valuable and easily adaptable BioBrick that can be used to detect any molecule for which an aptamer exists. In our case, the main goal is to validate the system and insert a PFOA-sensing aptamer, as illustrated in Figure X.</p>
        <h3 id="bioinformatics">Bioinformatics for better aptamer selection</h3>
        <p>In addition to <em>in vitro</em> validation, our plan also involves the development of a bioinformatics tool. The purpose of the tool is to streamline the design and optimization of aptamers by consolidating various molecular modeling tools used for aptamer dynamics into a single pipeline. By providing a comprehensive understanding of aptamer-analyte dynamics prior to laboratory experimentation, we anticipate that this tool will enhance the attractiveness of our molecular device.
        Usually, aptamers are obtained through the challenging and time-consuming process SELEX, as mentioned earlier. Unfortunately, SELEX is experimentally challenging, time consuming, and does not guarantee the discovery of the best-performing aptamer (Flamme et al. 2019, Zhu et al. 2019). To address this limitation, researchers have been exploring predictive tools that take a rational approach to identifying the most effective aptamer sequence for a given ligand (Emami et al. 2020).Notably, the Heidelberg iGEM team developed a tool called Making Aptamers Without SELEX (MAWS) in 2015, which was subsequently improved in 2017 (iGEM Heidelberg 2015, iGEM Heidelberg 2017). MAWS utilizes an entropic criterion for progressive selection of bases to predict aptamers in silico.
        However, there is still limited understanding of the conformational changes that occur in aptamers upon ligand binding. This knowledge is crucial for a comprehensive characterization of intracellular aptamer behavior and the refinement of models. Additionally, there is a lack of integrated pipelines that encompass all the steps involved in modeling aptamer structure prediction, docking, and molecular dynamics. To address these challenges, we developed an integrated pipeline, available through Docker, which generates and analyzes aptamers in an easy and accessible format. </p>
        <h3 id="cell_free">Designing a cell free system for the PFOA quick test</h3>
        <p>We hope that our iGEM project will be used as a steppingstone to create a cell free PFOA biosensor based on the technology developed by Guzman-Chavez et al. (2022). Cell-free technology utilizes <em>in vitro</em> transcription-translation systems in conjunction with cell extracts to provide ribosomes and transcription factors. Additionally, various other components are necessary for protein production. Although this technology has been financially out of reach for many applications, recent advancements offer promising prospects for significant cost reduction. These advancements encompass the utilization of more affordable energy sources, optimizing production with low-cost energy inputs, and simplifying the reaction mixture (Guzman-Chavez et al. 2022). Given our aim of providing an affordable PFAS rapid test, it is imperative to integrate it into a cost-effective cell-free system.</p>
        <h3 id="high_turnover">High-turnover environmental screening</h3>
        <p>With the goal of developing a cell-free detection system for PFAS quantification, we aim to give governments and NGOs the ability to screen for PFAS rapidly and cost-effectively. By paving the way for the creation of an affordable, fast, and user-friendly device, we aim to help aid environmental agencies gain insights into the prevalence and severity of PFAS in their localities. If PFAS testing is made rapid and cost effective, we believe global data generation will increase ten-fold, leading to better clean up, mitigation strategies, and public awareness, that will help build a better and safer world for us all. </p>
        <h3 id="conclusion">Conclusion</h3>
        <p>In conclusion, the persistent and hazardous nature of PFAS chemicals poses a significant threat to both human health and environmental stability, warranting immediate and effective detection measures. Current methodologies such as LC-MS are resource-intensive, time-consuming, and not universally accessible. To address this, our iGEM project, FluoroLoop, proposes a novel, modular biosensor based on tRNA-mimicking structures and PFOA-specific aptamers. This biosensor is designed to be both cost-effective and easily adaptable for various types of PFAS molecules. Our project also includes a bioinformatics tool to streamline aptamer design and a future plan for implementing this technology in a cell-free system for public use.</p>
        <p>Through FluoroLoop, we aim to democratize PFAS detection by developing a quick and affordable testing method that can be used globally. By empowering environmental agencies to monitor PFAS pollution in their environments, we hope to contribute significantly to the broader efforts to understand and mitigate the risks associated with these &#39;forever chemicals&#39;.</p>
        <h2 id="references">References</h2>
        <hr>
        <ol>
          <li>Brunn, H., Arnold, G., Körner, W., Rippen, G., Steinhäuser, K. G., &amp; Valentin, I. (2023). PFAS: forever chemicals—persistent, bioaccumulative and mobile. Reviewing the status and the need for their phase out and remediation of contaminated sites. Environmental Sciences Europe 2023 35:1, 35(1), 1–50. <a href="https://doi.org/10.1186/S12302-023-00721-8">https://doi.org/10.1186/S12302-023-00721-8</a> </li>
          <li>Cordner, A., Goldenman, G., Birnbaum, L. S., Brown, P., Miller, M. F., Mueller, R., Patton, S., Dambach, M., Sandoval, M., Updegrove, T. B., Anantharaman, V., Aravind, L., Waters, L. S., &amp; Storz, G. (2015). The ubiquitous yybP-ykoY riboswitch is a manganese-responsive regulatory element. Molecular Cell, 57(6), 1099–1109. <a href="https://doi.org/10.1016/j.molcel.2015.01.035">https://doi.org/10.1016/j.molcel.2015.01.035</a> </li>
          <li>Salvatore, D. H., &amp; Trasande, L. (2021). The True Cost of PFAS and the Benefits of Acting Now. Environmental Science and Technology, 55(14), 9630–9633. <a href="https://doi.org/10.1021/ACS.EST.1C03565/ASSET/IMAGES/LARGE/ES1C03565_0001.JPEG">https://doi.org/10.1021/ACS.EST.1C03565/ASSET/IMAGES/LARGE/ES1C03565_0001.JPEG</a> </li>
          <li>Suess, B., Fink, B., Berens, C., Stentz, R., &amp; Hillen, W. (2004). A theophylline responsive riboswitch based on helix slipping controls gene expression in vivo. Nucleic Acids Research, 32(4), 1610-1614. <a href="https://doi.org/10.1093/nar/gkh321">https://doi.org/10.1093/nar/gkh321</a> </li>
          <li>Emami, N., Pakchin, P. S., &amp; Ferdousi, R. (2020). Computational predictive approaches for interaction and structure of aptamers. Journal of Theoretical Biology, 497, 110268. <a href="https://doi.org/10.1016/J.JTBI.2020.110268">https://doi.org/10.1016/J.JTBI.2020.110268</a> </li>
          <li>Flamme, M., McKenzie, L. K., Sarac, I., &amp; Hollenstein, M. (2019). Chemical methods for the modification of RNA. Methods, 161, 64–82. <a href="https://doi.org/10.1016/J.YMETH.2019.03.018">https://doi.org/10.1016/J.YMETH.2019.03.018</a> </li>
          <li>Guzman-Chavez, F., Arce, A., Adhikari, A., Vadhin, S., Peoza-Garcia, J. A., Gandini, C., Ajioka, J. W., Molloy, J., Sanchez-Nieto, S., Varner, J. D., Federici, F., &amp; Haseloff, J. (2022). Constructing Cell-Free Expression Systems for Low-Cost Access. ACS Synthetic Biology, 11(3), 1114–1128. <a href="https://doi.org/10.1021/ACSSYNBIO.1C00342/SUPPL_FILE/SB1C00342_SI_002.XLSX">https://doi.org/10.1021/ACSSYNBIO.1C00342/SUPPL_FILE/SB1C00342_SI_002.XLSX</a> </li>
          <li>Park, J., Yang, K. A., Choi, Y., &amp; Choe, J. K. (2022). Novel ssDNA aptamer-based fluorescence sensor for perfluorooctanoic acid detection in water. Environment International, 158, 107000. <a href="https://doi.org/10.1016/J.ENVINT.2021.107000">https://doi.org/10.1016/J.ENVINT.2021.107000</a> </li>
          <li>Paul, A., Warszawik, E. M., Loznik, M., Boersma, A. J., &amp; Herrmann, A. (2020). Modular and Versatile Trans-Encoded Genetic Switches. Angewandte Chemie International Edition, 59(46), 20328–20332. <a href="https://doi.org/10.1002/ANIE.202001372">https://doi.org/10.1002/ANIE.202001372</a> </li>
          <li>Se interaktivt kort med de 104 største PFAS-forureninger af grundvandet | Indland | DR. (n.d.). Retrieved June 27, 2023, from <a href="https://www.dr.dk/nyheder/indland/se-interaktivt-">https://www.dr.dk/nyheder/indland/se-interaktivt-</a> kort-med-de-104-stoerste-pfas-forureninger-af-grundvandet </li>
          <li>Team:Heidelberg/software/maws - 2015.igem.org. (n.d.). Retrieved June 27, 2023, from <a href="https://2015.igem.org/Team:Heidelberg/software/maws">https://2015.igem.org/Team:Heidelberg/software/maws</a> </li>
          <li>Team:Heidelberg/Software/MAWS - 2017.igem.org. (n.d.). Retrieved June 27, 2023, from <a href="https://2017.igem.org/Team:Heidelberg/Software/MAWS">https://2017.igem.org/Team:Heidelberg/Software/MAWS</a> </li>
          <li>Zhu, C., Yang, G., Ghulam, M., Li, L., &amp; Qu, F. (2019). Evolution of multi-functional capillary electrophoresis for high-efficiency selection of aptamers. Biotechnology Advances, 37(8), 107432. <a href="https://doi.org/10.1016/J.BIOTECHADV.2019.107432">https://doi.org/10.1016/J.BIOTECHADV.2019.107432</a></li>
        </ol>
          

      </div>
    </div>
  </div>
</div>

{% endblock %}
